salt/modules/pillar.py

```diff
@@ -62,6 +63,24 @@ def get(key,
             then merging will be skipped and the retrieved value will be
             returned. Earlier releases raised an error in these cases.
 
+    deep_merge_lists : ``None``
```

Comment:
Well, what we could do to make this more intuitive is we could make the ``merge`` argument remove duplicates when ``default`` is a list. The code for that would be right here in ``pillar.get``.

We could add a blurb to the docstring to say "If ``default`` is a list, the retrieved value will be merged into it, otherwise it will replace the ``default`` value."